### ISSUE TITLE
How I Killed Payment Fees Forever…and How You Can, Too!

### ISSUE THREAD

**Issue by szwnba** [szwnba/affweb#137]
⚠️ **Note:** _Nothing in this thread should be considered financial advice. I am not a financial advisor. Do your own research._

There ain't no better day than payday. 🤑

Well… until you realize you didn't get paid as much as you thought you would get paid… thanks to those pesky payment processing fees.

If you didn't lose money when you got paid, you'll probably lose some when you cycle back to your [traffic source](https://afflift.com/f/link-directory/categories/traffic-sources.1/) to keep your campaigns running.

At the end of the year, you'll look back on how many thousands of dollars you paid on fees and throw up in your mouth.

Yes, there are some services, like Paxum, that have very low fixed fees. Still, depending on your situation (where you live, citizenship, financial, etc.), you may not be able to use services like these. Plus, not every affiliate network or traffic source will support Paxum (or similar)… forcing you to bring in another service and more fees.

In the last year, some changes to laws forced me to find a new bank, [ultimately leading me to find this one](https://cllick.link/bank) that helped me completely eliminate payment fees (and I actually make a profit by using it💰).

Today I will try and explain how your money flows around as an affiliate media buyer and what I've done to minimize/eliminate payment processor fees (and how you can do the same).

### The Money Flow​

As an affiliate media buyer, you have a lot of things to think about.

Not only do you have to keep up with your campaign launching and optimization regiment, but you also have to consider how you will keep all that funded.

It's a constant circle of dollar bills...

You get paid by your affiliate network to your account, which then has to be transferred to your traffic source to buy more traffic and fill up your affiliate network earnings again so the process can repeat.

When you move money, there's some kind of fee involved, which varies depending on the payment method(s) you're using.

These fees cut into your bottom line, so you must carefully consider your options and plan accordingly.

### **What Payment Methods are "Best"?**​

Since the available payment methods and fees vary greatly depending on where you live, I can't tell you the best payment method for your particular circumstance.

However, I suggest you use something popular and widely available on most traffic source(s) and affiliate network(s). This way, you can easily move your money around as needed without an extra transfer (usually involving even more fees).

### **Methods with % Based Fees**​

Payment methods with fees based on a percentage of your transfer amount (i.e., PayPal) can be an easy, reliable, and widely accepted way to move stacks around.

However, these methods are only good when moving around small amounts of money.

When you start moving larger amounts (usually around $500+ in a single transaction), the percentage-based fees are generally higher than fixed fees you'd have with payment methods like bank transfers.

#### **Fees in, Fees out**​

Sometimes with payment methods like PayPal, you'll get hit with fees to receive and send your money. Double fees... so instead of having a 3% fee, you now have a 6% fee.

**💡 Pro Tip:** _Sometimes, you can negotiate with your affiliate network or traffic source to pay the fees (sending or receiving). To get them to actually agree, it'll help if you're actually doing volume to make it worth it to them._

Make sure you're considering the whole picture when deciding on the best payment method for your situation.

### **Methods with Fixed Fees**​

Payment methods, like bank transfers, are widely accepted by just about everyone. Their main downsides are that they're not instant and might not be as cost-effective as other methods if you're only transferring tiny amounts ($100 ish).

However, if you're making payments and getting paid $500+ at a time, a bank transfer is probably one of the most cost-effective and most supported ways to get paid and make payments.

This definitely depends on your bank, though. My old bank would sometimes charge $80 - $100 to receive a wire transfer (even if the sender pays the fees on their end… talk about a scam, lol).

### How I Destroyed Payment Fees​

I live in a country where Euro is the official currency. So I need Euros to buy stuff here. However, I run my business primarily in USD for a few reasons:

1.  I'm from the US, and that's what I'm used to
2.  All affiliate networks and traffic sources I work with support this, and they don't all support Euro

Since my money is flowing continuously to fund campaigns, I have a [USD bank account with this bank](https://cllick.link/bank). It allows me to receive wire transfers from anyone for a fixed $4.14 fee (if outside the US), and if they're inside the US, I can receive ACH bank transfers for free. If I'm working with someone outside the US who has to send me wires for payment, I try to keep them monthly, so I only pay one fixed fee regardless of how much I get paid.

So receiving money costs me very little… if anything at all.

Sending money is also very cheap. I can send 20k USD (or any other amount) for a fixed $0.39, which would amount to hundreds of dollars with something like PayPal. Since this can take some time and the fee is fixed, I try to minimize the number of transfers I'll make in a month by planning ahead.

So sending money also costs me very little…

Being based in a country where Euro is the primary currency, I occasionally have bills to pay in Euro (taxes, business fees, accountant, etc.), so I also have a [Euro bank account at the same bank](https://cllick.link/bank). I can instantly do a currency conversion from my USD bank account to my Euro bank account (for typically one of the best rates you can get from any bank) and proceed to make a free Euro to Euro bank transfer to pay my bills. The same is true when I want to pay myself, but when I do this, I time the currency conversion market to maximize how much Euro I'll receive. 🤑

Sometimes I see people complaining about currency conversion fees, and it always makes me scratch my head. 🤔

Why do you receive USD, then convert it to another currency, only to send it back to the traffic source again (probably in USD)? It makes no sense, and you just lost a good % of your earnings doing that. So don't convert it unless you need it right now in that other currency...

So I keep most of my cash flow in USD… cycling it in and out of this bank account with the least amount of transfers I can to save on fees. By keeping my cashflow primarily as USD, I then receive cashback each month from my bank (as of the time of this writing, it's 3.64% for USD, 2.95% for GBP, and 1.59% for EUR), paying for any of the fees I was charged that month.

This cashback feature is new, but it's absolutely amazing. The interest rate is considerably higher than I had with a standard US bank. Just one month of cashback from this bank gave me more money back than I've got in my entire life from the interest rates banks usually pay!

In summary, by using [this bank](https://cllick.link/bank), I've ultimately eliminated payment fees. I try to make as few transfers as possible in a month that can result in fees when sending or receiving money, and then make those fees I was charged back from the high cashback percentage I get each month. With a higher balance in my account(s), this typically results in an end-of-month profit from the cashback rather than a loss from the payment processing fees. 😍

You can do the same by [signing up for a bank account with my link](https://cllick.link/bank) (which gives you one fee-free transfer of up to €500) and strategically managing your money so you don't make more transfers than are necessary.
